### PR TITLE
Admins can now spawn cosmetic blood stones

### PR DESCRIPTION
**Pull request by DeityLink**
The button is on _that_ panel below "Create a Turret" and "Create a new virus in a dish". When pressing it you get the choice between spawning them with no extras, or have them play their old spawn animation, featuring obsidian pillars and the destruction/cultification of their surroundings. They always appear fully filled. You can make them pulse by calling their set_animate() proc.

Cultists will automatically use the cultdance emote when they touch them, and they can be destroyed with no repercussions.

Also potentially fixed a bug that caused astral projections to kill their users.

### CODE REVIEW

**Comment by Eneocho** [comment]
Requesting for said adminbus-able blood stones to have selectable colours (so the admin could spawn a green "blood" stone).

File: code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm [hunk 1]
```diff
@@ -1319,6 +1325,155 @@ var/list/cult_spires = list()
 		if (3)
 			takeDamage(20)
 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//                       //Spawned from the Raise Structure rune. Available from Act II
```
Comment (DeityLink) [inline code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm]:
```suggestion
//                       //Re-added as a cosmetic structure by admin request
```

File: code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm [hunk 2]
```diff
@@ -1319,6 +1325,155 @@ var/list/cult_spires = list()
...
+//      CULT FORGE       //Also a source of heat
```
Comment (DeityLink) [inline code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm]:
```suggestion
//      BLOOD STONE      //
```

File: code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm [hunk 3]
```diff
@@ -1319,6 +1325,155 @@ var/list/cult_spires = list()
 		if (3)
 			takeDamage(20)
 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//                       //Re-added as a cosmetic structure by admin request
+//      CULT FORGE       //Also a source of heat
+//                       //
+///////////////////////////
+
+/obj/structure/cult/bloodstone // Re-added as a cosmetic structure by admin request
```
Comment (DeityLink) [inline code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm]:
```suggestion
/obj/structure/cult/bloodstone
```

File: code/modules/admin/topic.dm [hunk 4]
```diff
@@ -4776,7 +4790,8 @@
 			D.link_to_shuttle(shuttle_to_add_to)
 			to_chat(usr, "Assigned the [D] to [shuttle_to_add_to.name]")
 
-		message_admins("<span class='notice'>[key_name_admin(usr)] has created a new shuttle docking port in [get_area(D)] [formatJumpTo(get_turf(D))][shuttle_to_add_to ? " and assigned it to [shuttle_to_add_to.name]" : ""]</span>", 1)
+		message_admins("<span class='notice'>[key_name_admin(usr)] has created a new shuttle docking port in [get_area(D)] [format
+		To(get_turf(D))][shuttle_to_add_to ? " and assigned it to [shuttle_to_add_to.name]" : ""]</span>", 1)
```
Comment (DeityLink) [inline code/modules/admin/topic.dm]:
```suggestion
```

File: code/modules/admin/topic.dm [hunk 5]
```diff
@@ -4776,7 +4790,7 @@
 			D.link_to_shuttle(shuttle_to_add_to)
 			to_chat(usr, "Assigned the [D] to [shuttle_to_add_to.name]")
 
-		message_admins("<span class='notice'>[key_name_admin(usr)] has created a new shuttle docking port in [get_area(D)] [formatJumpTo(get_turf(D))][shuttle_to_add_to ? " and assigned it to [shuttle_to_add_to.name]" : ""]</span>", 1)
+		message_admins("<span class='notice'>[key_name_admin(usr)] has created a new shuttle docking port in [get_area(D)] [format
```
Comment (DeityLink) [inline code/modules/admin/topic.dm]:
```suggestion
		message_admins("<span class='notice'>[key_name_admin(usr)] has created a new shuttle docking port in [get_area(D)] [formatJumpTo(get_turf(D))][shuttle_to_add_to ? " and assigned it to [shuttle_to_add_to.name]" : ""]</span>", 1)
```

File: code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm [hunk 6]
```diff
@@ -1319,6 +1325,155 @@ var/list/cult_spires = list()
...
+		if (prob(5))
+			user.say("Let me show you the dance of my people!","C")
+		else
+			user.say("Tok-lyr rqa'nap g'lt-ulotf!","C")
+
+/obj/structure/cult/bloodstone/conceal()
+	return
+
+/obj/structure/cult/bloodstone/takeDamage(var/damage)
+	var/backup = (health > (2*maxHealth/3)) + (health > (maxHealth/3))
```
Comment (DeityLink) [inline code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm]:
```suggestion
```

**Pull request merged**